app/constants/user-types.js

```diff
@@ -1,8 +1,8 @@
 const farmerApply = 'farmerApply'
 
-const appliedBefore = {
+const userType = {
```

Comment:
I suggest this to be changed as user types because it contains different users within this object